sdk/nodejs/cmd/pulumi-language-nodejs/main.go

```diff
@@ -1378,7 +1378,7 @@ func (host *nodeLanguageHost) RunPlugin(
 		return err
 	}
 
-	env := os.Environ()
+	env := req.Env
```

Comment:
The changes in this file look to be unrelated

Reply:
Yeah, let me pull these out.

Reply:
https://github.com/pulumi/pulumi/pull/17836